components/script/dom/attr.rs

```diff
@@ -264,17 +256,17 @@ impl<'a> AttrHelpers<'a> for &'a Attr {
         }
     }
 
-    fn value(self) -> Ref<'a, AttrValue> {
+    pub fn value<'a>(&'a self) -> Ref<'a, AttrValue> {
```

Comment:
No need for using 'a anywhere in this line (`fn foo(&self)->Ref<Bar>` will work) because elision.

ditto everywhere else in this PR